numpy/array_api/_array_object.py

```diff
@@ -124,7 +125,7 @@ def __array__(self, dtype: None | np.dtype[Any] = None) -> npt.NDArray[Any]:
     # spec in places where it either deviates from or is more strict than
     # NumPy behavior
 
-    def _check_allowed_dtypes(self, other, dtype_category, op):
+    def _check_allowed_dtypes(self, other: bool | int | float | Array, dtype_category: str, op: str) -> Array:
```

Comment:
Is there a way to get this line down to 80 columns or should we leave it as-is?

Reply:
please leave it as is. this came up in review of Aaron's main PR too, wrapping at 80 char would have made things much less readable.

also, I think to avoid this coming up over and over, I think we should have a separate CI config setting of 88 char (while staying with the default for local development). Review questions for "this is 82 char, should be <79" are almost never useful.

Reply:
or maybe even 100 char, I dunno.

Reply:
Ok, but this one is 110, which is a little extraordinary even for wide screens.